indico/htdocs/js/indico/modules/timetable/timetable/Draw.js

```diff
@@ -1958,25 +1958,102 @@ type("IntervalTimetableDrawer", ["TimetableDrawer"],
             var topPx = 0;
             each(data, function(blockData, id) {
 
-                var editLink = Html.a({className: 'dropDownMenu', style: {fontWeight: 'bold'}}, $T('Edit'));
+                function createEditLink(blockData) {
+                    var editLink = Html.a();
+                    editLink.dom = $('<a>', {
+                        class: 'icon-edit i-button-icon',
+                        title: $T.gettext("Edit"),
+                        data: {
+                            title: $T.gettext("Edit poster"),
+                            confId: blockData.conferenceId[0],
+                            timetableEntryId: blockData.scheduleEntryId
+                        }
+                    })[0];
+
+                    $(editLink.dom).on('click', function(evt, params) {
+                        var $this = $(this);
+                        var params = {
+                           'confId': $this.data('confId'),
+                           'timetable_entry_id': $this.data('timetableEntryId')
+                        };
+                        ajaxDialog({
+                            trigger: this,
+                            url: build_url(Indico.Urls.Timetable.entries.edit, params),
+                            title: $this.data('title'),
+                            onClose: function(data) {
+                                if (data && data.entries) {
+                                    self.managementActions._addEntries(data.entries);
+                                }
+                            }
+                        });
+                    });
+
+                    return editLink;
+                }
+
+                function createProtectionLink(blockData) {
+                    var protectionLink = Html.a();
+                    protectionLink.dom = $('<a>', {
+                        class: 'icon-shield i-button-icon',
+                        title: $T.gettext("Manage protection"),
+                        data: {
+                            title: $T.gettext("Manage poster protection"),
+                            confId: blockData.conferenceId[0],
+                            contribId: blockData.contributionId
+                        }
+                    })[0];
+
+                    $(protectionLink.dom).on('click', function(evt) {
```

Comment:
just assign the jquery object somewhere so you don't have to go from jquery to legacy and back to jquery